lib/src/core/guild/guild_preview.dart

```diff
@@ -39,15 +38,15 @@ abstract class IGuildPreview implements SnowflakeEntity {
 
   /// The guild's icon, represented as URL.
   /// If guild doesn't have icon it returns null.
-  String? iconURL({String format = "webp", int size = 128});
+  String? iconUrl({String? format, int? size, bool animatable = false});
```

Comment:
Hmm, I don't know, I chose `animatable` because that would something to cause some conditional effects.
But why not, it's maybe more self-explanatory than this current wording

Reply:
yeah, I feel like it would be better that way

Reply:
Then it's fine, I'll rename these props when i'll have the time